src/engine/maps/GuidoMapCollector.cpp

```diff
@@ -117,6 +117,7 @@ void GuidoVoiceAndBarCollector::Graph2TimeMap( const FloatRect& box, const TimeS
 {
     bool isBar = (infos.type == kBar || infos.type == kRepeatBegin || infos.type == kRepeatEnd);
     
+    if ( fStaffNum != 0 && fStaffNum != infos.staffNum)   return; // other staves are filtered out if fStaffNum is defined
```

Comment:
you should use parenthesis for the second part of the test:
if ( fStaffNum != 0 && fStaffNum != infos.staffNum)
it's not for safety but for readability (I know the precedence rule will apply)